### PR TITLE
Rank the address search

### PR DESCRIPTION
**Pull request by nishadhperera**

### CODE REVIEW

File: App/src/main/java/cc/softwarefactory/lokki/android/activities/SearchActivity.java [hunk 1]
```diff
@@ -106,7 +113,7 @@ protected void onCreate(Bundle savedInstanceState) {
         Log.d(TAG, "User searched for: " + queryMessage);
 
         resultList = new ArrayList<>();
-        setHeader(queryMessage);
+        //setHeader(queryMessage);
```
Comment (hassanabbas9) [inline App/src/main/java/cc/softwarefactory/lokki/android/activities/SearchActivity.java]: Please remove the unnecessary lines.

File: App/src/main/java/cc/softwarefactory/lokki/android/activities/SearchActivity.java [hunk 2]
```diff
@@ -229,7 +236,12 @@ protected void searchPlaces(String query, ArrayList<SearchResult> resultList){
      */
     private void searchGoogleMaps(final String query){
         final AQuery aq = new AQuery(this);
-        String url = GOOGLE_MAPS_API_URL + query;
+        UserLocation loc = MainApplication.user.getLocation();
+        Double lon = loc.getLon();
+        Double lat = loc.getLat();
+
+        String url = GOOGLE_MAPS_API_URL+ query +"&location="+lat+","+lon+"&radius="+radius+"&key="+ API_Key;
```
Comment (hassanabbas9) [inline App/src/main/java/cc/softwarefactory/lokki/android/activities/SearchActivity.java]: Please provide consist spaces in the url before and after the + sign.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by nishadhperera** [comment]
Please rebuild this

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by nishadhperera** [comment]
Please rebuild this

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests failed.

**Comment by SoftwareFactoryAutomation** [comment]
Tests passed. All is well.

**Pull request merged**